csrc/python_frontend/python_bindings.cpp

```diff
@@ -2726,6 +2735,48 @@ void initNvFuserPythonBindings(PyObject* module) {
   py::class_<FusionDefinition::SchedOperators> nvf_sched(
       fusion_def, "SchedOperators");
   nvf_sched.def(py::init<FusionDefinition*>());
+  //! experimental API for multidevice support
+  nvf_sched.def(
+      "_create_device_mesh",
+      [](FusionDefinition::SchedOperators& self, std::vector<int64_t> devices) {
+        FUSER_PERF_SCOPE("SchedOperators._create_device_mesh");
+        return DeviceMesh(devices);
+      },
+      py::arg("devices"),
+      py::return_value_policy::reference);
+  //! experimental API for multidevice support
+  nvf_sched.def(
+      "_set_device_mesh",
+      [](FusionDefinition::SchedOperators& self, Tensor arg, DeviceMesh mesh) {
```

Comment:
I saw many APIs in this file take values instead of references. Is that intentional? IOW, can we do `const DeviceMesh& mesh` here?